frontend/app/src/components/history/events/HistoryEventsView.vue

```diff
@@ -18,6 +20,7 @@ import HistoryEventsTable from '@/modules/history/events/components/HistoryEvent
 import { useHistoryEventsDeletion } from '@/modules/history/events/composables/use-history-events-deletion';
 import { useHistoryEventsSelectionMode } from '@/modules/history/events/composables/use-selection-mode';
 import { useHistoryEventsStatus } from '@/modules/history/events/use-history-events-status';
+import { useConfirmStore } from '@/store/confirm';
```

Comment:
Any way to keep the imports here at max 20 without suppressing eslint?
Maybe we could also move the accounting rules specific code to it's own composable?

And the dialog should go into the `HistoryEventsDialogContainer`